templates/assessor_manager/_layout.html.twig

```diff
@@ -59,7 +59,7 @@
 
                 <ul class="assessor-manager__tabs">
                     {% for page in pages %}
-                        <li class="{{ app.request.attributes.get('_route') == page.route and filters.status == page.status ? 'active' : '' }}">
+                        <li class="{{ is_active_route(app.request, page.route) and filters.status == page.status ? 'active' }}">
```

Comment:
Just asking, the `: ''` is not a mandatory ?

Reply:
yes, it's optional, I discovered this yesterday 😄